packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -58,15 +49,16 @@ import {
 
 import { importResource } from '@cardstack/host/resources/import';
 
+import {
+  inThisFileResource,
```

Comment:
we talked about a better name here. please make sure to change that before merging

Reply:
Fixed here https://github.com/cardstack/boxel/pull/716/commits/380cf48e1d2e1119804a4ef8b3bcab0fea8f7036

Reply:
- InThisFileResource -> ModuleContentsResource
- elements -> declarations